components/NewsletterForm.tsx

```diff
@@ -35,47 +37,179 @@ const Container = styled.div`
     width: 100%;
   }
 `
+const NewsletterForm = ( { status, message, onValidated }) => {
 
-// TO DO: Fix button size on desktop
+  const [ error, setError ] = useState(null)
+  const [ email, setEmail ] = useState(null)
 
-const NewsletterForm = () => {
+  /**
+   * Handle form submit.
+   *
+   * @return {{value}|*|boolean|null}
+   */
+  const handleFormSubmit = () => {
 
-  const [form, setForm] = useState('')
-  const [validateEmail, setValidateEmail] = useState('')
+    setError(null)
 
-  const handleChange = (e: React.FormEvent<HTMLInputElement>) => {
-    setValidateEmail('')
-    setForm(e.currentTarget.value)
-  }
+    if ( ! email ) {
+      setError( 'Vänligen fyll i en giltig e-postadress.' )
+      return null
+    }
 
-  const handleSubmit = (e: any) => {
+    const isFormValidated = onValidated({ EMAIL: email })
```

Comment:
Får följande error vid submit av formuläret:

![Screenshot 2022-02-23 at 12 17 59](https://user-images.githubusercontent.com/72319394/155309346-fb33c3ae-0801-4496-ac5d-b5d0d41c54da.png)

Reply:
Det är inte ett typescript-error, du skickar inte in onValidated till denna komponent